src/main/java/hudson/plugins/gradle/injection/maven/GradleEnterpriseExtensionsContributorFactory.java

```diff
@@ -44,6 +49,15 @@ public PlexusModuleContributor createFor(AbstractBuild<?, ?> build) {
                 return EMPTY_CONTRIBUTOR;
             }
 
+            if (!isSupportedMavenPluginVersion()) {
+                LOGGER.log(
+                    Level.WARNING,
+                    "An older maven-plugin version is used. Please use at least the version {0} for auto-injection to work",
```

Comment:
Can we list the current Maven plugin version ?

I'd then also write it that way:

```
Detected Maven Integration plugin version x.x.x. For auto-injection of the Gradle Enterprise Maven extensions, version x.x.x is required. Please upgrade the version of the Maven Integration plugin.
```

Reply:
Done